Src/FluentAssertions.Json/JTokenDifferentiator.cs

```diff
@@ -304,9 +304,9 @@ public override string ToString()
                 DifferenceKind.ExpectedMissesProperty => $"has extra property {Path}",
                 DifferenceKind.ActualMissesElement => $"misses expected element {Path}",
                 DifferenceKind.WrongOrder => $"has expected element {Path} in the wrong order",
-#pragma warning disable S3928 // Parameter names used into ArgumentException constructors should match an existing one 
+#pragma warning disable S3928, MA0015 // Parameter names used into ArgumentException constructors should match an existing one 
```

Comment:
⛏️When I see that two different analyzers report the same thing, I'll disable one in the `.editorconfig` like I did in https://github.com/fluentassertions/fluentassertions/blob/main/.editorconfig#L403